### ISSUE TITLE
Multi-stage deals

### ISSUE THREAD

**Issue by ScatteredInk** [SocialInvestmentDataLab/spec#15]
In order to fully describe leverage and externality in social investment, there is a use-case for analysing deals as a multi-stage or multi-part process.

For example:

1.   `DonorOrg1` issues a business development grant of £10K and a £100K loan to `RecipientOrg1` at time **t<sub>0</sub>**.
2. The business development is a success and `DonorOrg1` then offers match funding of £100K on a community share issue. The equity offer raises an additional £200K through from its public offering at time **t<sub>1</sub>**.
3. As a result of the sounder capital and business situation, a commercial bank `RegisteredEntity1` offers a loan of £500K at time **t<sub>2</sub>**.

Deals 1 and 2 could be considered as separate deals, and deal 3 could be considered out of scope of the specification altogether. But to fully capture the impact of the investments made at time **t<sub>0</sub>** and **t<sub>1</sub>**, funding organisations need the ability to:

1. Consider what happens _next_.
2. Analyse the stages of the deal separately.

In my ad-hoc analyses, I have simply been assigning a `deal part` identifier to disaggregate deals within larger deals. But, for a specification, this does bring up the thorny issue of what are the criteria for _stopping_ attribution of further investment to a particular deal?

**Comment by ScatteredInk** [comment]
I'm increasingly of the opinion that we _shouldn't_ model this in the specification because it is contingent on the reporting requirements of individual funders and data publishers. The potential to increase complexity is huge, while the analytical gains are relatively small. This kind of thing can probably be dealt with in an ad hoc manner (and, with structured data, relatively painlessly).

Worth canvassing opinions on but I think we need to be mindful of getting data published.

**Comment by BobHarper1** [comment]
I broadly agree. I think this, if it were to be addressed, would be contingent on scaling up both the level of data publication as well as the aptitude of those gathering the data. And it would also need to be a  relatively simple solution.

There is some link in your example between the financing, but I'm not convinced that they're contingent enough to justify developing a model for. No sound financier would lend/grant to a business without considering their ability to repay etc, so in theory any past deals can have a leveraging impact on the prospect of future deals, even if not inextricably linked.

If I were examining it, so long as I can see the correct data, I can make assumptions about contingent deals and the leverage involved. And if it can't provide all parts of a multi-stage deal (i.e. Stage 3) then the value of making a provision to do that is reduced. 

So I think the issue for the Spec is around developing guidance around what to do in such cases, and to publish those stages as separate deals.
